platforms/platform-alexa/src/output/templates/ApiResponseOutput.ts

```diff
@@ -0,0 +1,25 @@
+import { BaseOutput, Output, OutputOptions, OutputTemplate } from '@jovotech/framework';
+
+export interface ApiResponseOutputOptions extends OutputOptions {
+  apiResponse: Record<string, string | number | boolean>;
+}
+
+@Output()
+export class ApiResponseOutput extends BaseOutput<ApiResponseOutputOptions> {
+  build(): OutputTemplate | OutputTemplate[] {
+    return {
+      listen: this.options.listen,
```

Comment:
`listen` is a [reserved property](https://www.jovo.tech/docs/output-classes#reserved-properties), so it doesn't necessarily need to be added to this output. By default, it's `true`

Reply:
never mind, I think this is OK and clearer if we keep it in there